pombola/hansard/api/views.py

```diff
@@ -0,0 +1,59 @@
+from django.db.models import Prefetch
+from django.shortcuts import get_object_or_404
+
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from ..models import Entry, Sitting, Source, Venue
+from .serializers import (
+    EntrySerializer, SittingSerializer, SittingWithEntriesSerializer,
+    SourceSerializer, VenueSerializer
+)
+
+
+class EntryViewSet(viewsets.ModelViewSet):
+    queryset = Entry.objects.order_by('id')
+    serializer_class = EntrySerializer
+
+
+class SittingViewSet(viewsets.GenericViewSet):
+
+    queryset = Entry.objects.order_by('id')
```

Comment:
Sorry, I should have been more specific. It was the fact that it was in the `SittingViewSet` but the queryset is for `Entry` which was confusing about it.